.github/workflows/build-test.yaml

```diff
@@ -278,7 +278,6 @@ jobs:
       - name: Checkout gcc hash
         run: |
           cd gcc
-          git fetch
```

Comment:
Good catch - for my own future reference this is legal since we checkout the hash before uploading the cache artifact.